duckduckgo/duckduckgo-autofill
# New subdomain matching rules

**Reviewer:** @alistairjcbrown 
**Asana:** https://app.asana.com/0/0/1204349300101950/f
Needs macOS branch `autofill/anya/subdomain-api-updates`: https://github.com/duckduckgo/macos-browser/pull/1096

## Description
- The API now includes `origin` for credentials
- Show different label when entries come from a different subdomain
- Style tweaks:
  - Different hover color
  - Different hover border radius
  - Different font
  - Smaller spacing between the login labels
  - Smaller icon
- Other minor fixes

## Steps to test
Added automated tests. To test on device, make sure you use the macOS branch referenced above and follow the instructions in the [ship review](https://app.asana.com/0/1142021229838617/1204337437827825/f).
